apps/src/applab/designMode.js

```diff
@@ -250,6 +250,10 @@ designMode.updateProperty = function (element, name, value) {
       break;
     case 'icon-color':
       element.setAttribute('data-icon-color', value);
+      const imageUrl = element.getAttribute('data-canonical-image-url');
+      if (ICON_PREFIX_REGEX.test(imageUrl)) {
+        element.src = assetPrefix.renderIconToString(imageUrl, element);
```

Comment:
I see that in other usages of `renderIconToString` we sometimes assign the result to `src` as here, and sometimes to `backgroundImage` - it might depend on the type of element we're updating.  I'm still working out whether that means there's a problem case here, I'll try and track down a specific example.